components/mpas-ocean/src/analysis_members/Registry_eddy_product_variables.xml

```diff
@@ -83,5 +104,10 @@
 			<var name="velocityMeridionalTimesTemperature_GM"/>
 			<var name="normalVelocityTimesTemperature"/>
 			<var name="normalVelocitySquared"/>
+			<var name="velocityZonalTimesSalinity"/>
+			<var name="velocityMeridionalTimesSalinity"/>
+			<var name="normalVelocityTimesSalinity"/>
+			<var name="velocityZonalTimesSalinity_GM"/>
+			<var name="velocityMeridionalTimesSalinity_GM"/>
```

Comment:
Is there a reason to output both the `velocityZonalX` `velocityMeridionalX` and `normalVelocityX`?  I would guess we really only need one set or the other.

Reply:
Note also that to get these variables output in runs, changes are needed to 
```
components/mpas-ocean/cime_config/bldnml
```

Reply:
I think that historically we output both on centers and edges, to give people a choice between a quick visualization (centers) or a transect calculation (on edge). If the goal is to close budgets, I'd probably prioritize `normalVelocityX` but maybe that's just me.

Reply:
I think that is fair for normalVelocity and velocityZonal, velocityMeridional  and am not opposed to including both, but only if we actually think both will be useful.  My suggestion would be to keep both calculations, but only output the cell based quantities for eddy stats by default.  the edge quantities could be turned on for special simulations

Reply:
@vanroekel I added the zonal and meridional components in the default, with the edge quantity calculated but not in the default output, as you suggested above.